bundles/org.openhab.ui/web/src/pages/settings/rules/script/script-edit.vue

```diff
@@ -396,7 +396,13 @@ export default {
       return this.languages.map(l => l.contentType).includes(mimeType)
     },
     mimeTypeDescription (mode) {
-      return AddonNames[mode] || mode
+      return AUTOMATION_LANGUAGES[mode]['name'] || mode
+    },
+    documentationLink (mode) {
+      if ('documentationLink' in AUTOMATION_LANGUAGES[mode]) {
+        return AUTOMATION_LANGUAGES[mode]['documentationLink']
+      }
+      return null
```

Comment:
```suggestion
      return AUTOMATION_LANGUAGES[mode].documentationLink
```

Reply:
This will return `undefined` instead of `null`, which however shouldn‘t be a problem when checking for the return value without a comparison operator, e.g. `if (someValue) do something` or `if (!someValue) do nothing`.

Reply:
Without the check if documentationLink exists, I get the following error in Chrome: TypeError: Cannot read properties of undefined (reading 'documentationLink')

Reply:
this works if I use this form: return AUTOMATION_LANGUAGES[mode]['documentationLink']